pkg/ddc/jindo/delete_volume.go

```diff
@@ -1,5 +1,49 @@
 package jindo
 
+import (
+	volumeHelper "github.com/fluid-cloudnative/fluid/pkg/utils/dataset/volume"
+)
+
+// DeleteVolume creates volume
```

Comment:
DeleteVolume delete Fuse PersistentVolumeClaim and PersistentVolume

Reply:
fix